### PR TITLE
Migrate matrix renderer test to utest

### LINKED ISSUE #121: Set up utest
We should have real tests running. There is a script in `test` that can be converted to a utest.

### PR DESCRIPTION
**Pull request by Shalsh23**
Fixes #121

### CODE REVIEW

**Comment by balhoff** [comment]
You will need to update your branch from `master` for the travis checks to work correctly.

File: src/test/scala/org/phenoscape/kb/MatrixRendererTest.scala [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-  val innerMap2: Map[Any, Boolean] = Map("femur" -> true, "hindlimb" -> true, "forelimb" -> false, "hand" -> false, "face" -> false, "eye" -> false)
-  val innerMap3: Map[Any, Boolean] = Map("femur" -> false, "hindlimb" -> false, "forelimb" -> true, "hand" -> false, "face" -> false, "eye" -> false)
-  val innerMap4: Map[Any, Boolean] = Map("femur" -> false, "hindlimb" -> false, "forelimb" -> true, "hand" -> true, "face" -> false, "eye" -> false)
-  val innerMap5: Map[Any, Boolean] = Map("femur" -> false, "hindlimb" -> false, "forelimb" -> false, "hand" -> false, "face" -> true, "eye" -> true)
-  val innerMap6: Map[Any, Boolean] = Map("femur" -> false, "hindlimb" -> false, "forelimb" -> false, "hand" -> false, "face" -> false, "eye" -> true)
 
+object MatrixRendererTest extends TestSuite {
 
-  val outerMap: Map[Any, Map[Any, Boolean]] = Map("eye" -> innerMap6, "femur" -> innerMap1, "hindlimb" -> innerMap2, "forelimb" -> innerMap3, "hand" -> innerMap4, "face" -> innerMap5) //("y2" -> innerMap, "y3" -> innerMap, "y1" -> innerMap)
+  val tests = Tests{
```
Comment (balhoff) [inline src/test/scala/org/phenoscape/kb/MatrixRendererTest.scala]: Can you check the formatting settings in your IntelliJ? Should be a space in front of the brace here.

File: src/test/scala/org/phenoscape/kb/MatrixRendererTest.scala [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
+    val innerMap5: Map[Any, Boolean] = Map("femur" -> false, "hindlimb" -> false, "forelimb" -> false, "hand" -> false, "face" -> true, "eye" -> true)
+    val innerMap6: Map[Any, Boolean] = Map("femur" -> false, "hindlimb" -> false, "forelimb" -> false, "hand" -> false, "face" -> false, "eye" -> true)
 
-  println(outputString)
 
+    val outerMap: Map[Any, Map[Any, Boolean]] = Map("eye" -> innerMap6, "femur" -> innerMap1, "hindlimb" -> innerMap2, "forelimb" -> innerMap3, "hand" -> innerMap4, "face" -> innerMap5) //("y2" -> innerMap, "y3" -> innerMap, "y1" -> innerMap)
+
+    'testDep - {
+      val outputString = AnatomicalEntity.matrixRendererFromMapOfMaps(DependencyMatrix(outerMap, List("femur", "hindlimb", "forelimb", "hand", "face", "eye")))
+      println(outputString)
```
Comment (balhoff) [inline src/test/scala/org/phenoscape/kb/MatrixRendererTest.scala]: Since you're turning this into a test, go ahead and test something! :-)

**Comment by Shalsh23** [comment]
@balhoff Ready for merge.

**Pull request merged**